app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/Project.scala

```diff
@@ -63,8 +76,26 @@ object Project extends GeoJsonSupport {
         description,
         visibility,
         tileVisibility,
+        isAOIProject,
+        aoiCadenceMillis,
+        new Timestamp(now.getTime - aoiCadenceMillis),
         tags
       )
     }
   }
+
+  object Create {
+    /** Custon Circe decoder for [[Create]], to handle default values. */
+    implicit val projectDecoder: Decoder[Create] = Decoder.instance(c =>
+      (c.downField("organizationId").as[UUID]
+        |@| c.downField("name").as[String]
```

Comment:
I previously had:
```scala
for {
  o <- c.downField("organizationId").as[UUID]
  n <- c.downField("name").as[String]
  ...
} yield Create(o, n, ...)
```
but that was ugly as hell (and also Monadic (`flatMap`y), which isn't strictly necessary here), so I switched to the Applicative (`|@|`) form. Decoding Json is a `Functor`y operation, where each step doesn't depend on any other. We can apply a pure function to the aggregate results of it via `.map`.

Minimal example:
```scala
val a: Option[Int]
val b: Option[Int]
val c: Option[Int]

val res: Option[Int] = (a |@| b |@| c).map(_ + _ + _)
```